platform-bukkit/src/main/java/net/kyori/adventure/platform/bukkit/CraftBukkitAccess.java

```diff
@@ -85,12 +85,20 @@ final class CraftBukkitAccess {
     findMcClassName("core.IRegistryWritable"),
     findMcClassName("core.WritableRegistry")
   );
+  static final @Nullable MethodHandle NEW_RESOURCE_LOCATION;
+
+  static {
+    MethodHandle newResourceLocation = findConstructor(CLASS_RESOURCE_LOCATION, String.class, String.class);
+    if (newResourceLocation == null) {
+      newResourceLocation = searchMethod(CLASS_RESOURCE_LOCATION, Modifier.PUBLIC | Modifier.STATIC, "fromNamespaceAndPath", CLASS_RESOURCE_LOCATION, String.class, String.class);
```

Comment:
do we need to care about reobf here?

Reply:
there are 2 methods with a matching signature, the only difference is error handling (one throws, the other returns null), so the name doesn't matter - tested on mojmap paper 1.21 and reobf spigot 1.21